ekncontent/ekncontent/eknc-subtree-dispatcher.c

```diff
@@ -191,12 +202,20 @@ const GDBusSubtreeVTable subtree_vtable = {
   .dispatch   = subtree_dispatch,
 };
 
+/**
+ * eknc_subtree_dispatcher_register:
+ * @dispatcher: the subtree dispatcher
+ * @connection: the dbus connection
+ * @subtree_path: a subtree path to register with
+ *
+ * Register the dispatcher.
+ */
 void
-ekns_subtree_dispatcher_register (EknsSubtreeDispatcher *dispatcher,
+eknc_subtree_dispatcher_register (EkncSubtreeDispatcher *dispatcher,
```

Comment:
NIT, @ptomato should we use *self here instead of dispatcher

BTW I personally prefer dispatcher but it is my understanding that all our libs should use 'self'